draft-thoji-scone-protocol.md

```diff
@@ -145,22 +155,22 @@ congestion controller.
 
 ## Unspecified Scope
 
-Modifying a packet does not prove that the rate limit that is indicated would be
+Modifying a packet does not prove that the throughput that is indicated would be
 achievable.  A signal that is sent for a specific flow is likely enforced at a
 different scope.  The extent of that scope is not carried in the signal.
 
 For instance, limits might apply at a network subscription level, such
 that multiple flows receive the same signal.
 
-Endpoints can therefore be more confident in the rate limit signal as an
+Endpoints can therefore be more confident in the throughput signal as an
 indication of the maximum achievable throughput than as any indication of
-expected throughput.  That throughput will only be achievable when there is no
-significant data flowing in the same scope.  In the presence of other flows,
-congestion limits are likely to determine actual throughput.
+expected throughput.  Where the throughput signal simply represents a rate
+limit, then the advised throughput will only be achievable on an uncongested network.
+Whereas a throughput signal calculated by a non rate-limiting network should be 
+based on current network and attached device signal conditions, and hence account for
+any network congestion.
```

Comment:
I don't like this one.  The idea that this mechanism is good for congestion feedback is not something that has broad support.  That's why ECN or similar mechanisms are for.

Reply:
Reverted (except I have kept 'throughput' in place of the original 'rate limit')